src/views/Point.js

```diff
@@ -48,6 +51,30 @@ export default function Point() {
     history,
   ]);
 
+  const inviteButton = useMemo(() => {
```

Comment:
you'll want to raise all of those callbacks out of this function, and there's no real need to useMemo here.

```jsx
const goInvite = useCallback(() => history.push(ROUTE_NAMES.INVITE), [push, names]);
const goParties = useCallback(() => history.push(ROUTE_NAMES.INVITES_MANAGE), [push, names]);

const inviteButton = (() => {
    switch (azimuth.getPointSize(point)) {
      case azimuth.PointSize.Planet:
        return (
          <FooterButton disabled={!isOwner} onClick={goInvite}>
            Invite <sup>{matchBlinky(availableInvites)} available</sup>
          </FooterButton>
        );
      case azimuth.PointSize.Star:
        return (
          <FooterButton disabled={!isOwner} onClick={goParties}>
            Manage parties
          </FooterButton>
        );
      default:
        return null;
    }
})()
```

also please use the `names` object returned from `useHistory` so we can avoid pulling in `routeNames` everywhere.